assets/texts/fr/fr.txt

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
The file name is wrong. It has to be `messages.json`. Then, in a proper text editor (just use something simple like Atom or Notepad++ on Windows, installing takes like 5 minutes), you also get syntax highlighting.

See https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/i18n